webui/react/src/types.ts

```diff
@@ -111,7 +111,7 @@ export interface Resource {
 export interface Agent {
   id: string;
   registeredTime: number;
-  resourcePool: string;
+  resourcePools: string[];
```

Comment:
I think pulling the bandaid off is probably ok, so long as it isn't used anywhere, so long as when people upgrade they refresh to get the new UI?

Reply:
yeah the only time it would break is if the cluster version that is release after this change attempts to talk to a cluster before this change, then it wouldn't work...